src/TicketMachine.java

```diff
@@ -75,18 +104,102 @@ Ticket chooseFilm(){ //Ticket is the return type here. It is a Class Return and
         int choice = userInput.nextInt();
         switch (choice){
             case 1: //SW4, only during testing
-                return TL1.allFilms.get(0);
+                return availableTicketsList.get(0);
             case 2: //IJ1, only during testing
-                return TL1.allFilms.get(1);
+                return availableTicketsList.get(1);
             default:
                 throw new IllegalStateException("Unexpected value: " + choice); //IDEA gave me this. I like it :)
         } //Can I write a try/catch for this throw? How does that work? TODO figure out "throw try catch exceptions"
```

Comment:
Yes, you could. For instance: surround the 'main' part of the programme with a `try catch` and there you can handle the error. 
e.g. 
```
try {
  // do something
} catch (Exception e) {
  // log error
  // return user-friendly message
  // exit? restart?
}

Reply:
Ooh, the Log error is something to keep in mind. Thanks.

'main' part means 'main part of the code where the error will occur? So in this case surround the switch?
Or do you mean all the code that runs in main()?

Reply:
Could be either of them. In general it's good to have some top-level error handling, so that things can be cleaned up before exiting (e.g. closing resources). 
And you can also have a specific try-catch in this case surrounding the switch, in case you expect an exception and you want to handle it in a specific way.